lib/datadog/security/waf.rb

```diff
@@ -0,0 +1,68 @@
+require 'libsqreen'
+require 'datadog/security/assets'
+
+module Datadog
+  module Security
+    module WAF
+      class Args < Hash; end
+
+      class Rules < Hash; end
+
+      module_function
+
+      # TODO: logger
```

Comment:
Ok, there are enough loggers around here 😄, so I'd like to ask: why not use something like https://github.com/DataDog/dd-trace-rb/blob/f7431ce273cea753a8bee4f6e64ed2b2b098b04b/lib/ddtrace/configuration/settings.rb#L126-L132?

Reply:
I'm still getting to grips with the config/settings stuff, I appreciate the example!